src/shared/components/FormComponents/EditorController/EditorController.tsx

```diff
@@ -62,6 +62,7 @@ export const EditorController = <T extends FieldValues>({
         render={({ field: { onChange, value }, fieldState: { error } }) => (
           <StyledFlexColumn sx={{ position: 'relative' }} data-testid={dataTestid}>
             <StyledMdEditor
+              sanitize={(content: string) => getSanitizedContent(content)}
```

Comment:
could you check this suggestion, doesn't it throw TS error/warning?
```suggestion
              sanitize={getSanitizedContent}
```

Reply:
yes it throws because of other getSanitizedContent function arguments
that's why, I'm using this approach